src/track/serato/tags.h

```diff
@@ -41,7 +44,7 @@ class SeratoTags final {
         return m_seratoMarkers2.dump();
     }
 
-    QList<CueInfo> getCues(const QString& filePath) const;
+    CueInfoImporterPointer getCueInfoImporter() const;
```

Comment:
"get" doesn't fit. You are actually importing the cue points here, i.e. importCueInfos(). Only the result is evaluated lazily, like a promise/future. You have to provide the sample rate as the key to unlock the results.